nncf/torch/quantization/strip.py

```diff
@@ -163,14 +178,159 @@ def remove_disabled_quantizers(model: NNCFNetwork) -> NNCFNetwork:
+    while the latter takes a quantized integer representation, dequantizes it, and outputs a floating-point result.
+
+    Mathematically, both methods lead to the same outcome, but due to differences in the order of operations and
+    rounding errors, the actual results may differ. In particular, this error can occur for values
+    that are located in the midpoint between two quantized values ("quants").
+
+    The FQ format may round these values to one "quant", while the DQ format rounds them to another "quant".
+    To avoid these issues, the compressed representation should be provided not by directly quantizing the input,
+    but by quantizing a pre-processed, fake-quantized, floating-point representation.
+
+    :param model: Compressed model with Decompressors.
+    :return: The modified NNCF network.
+    """
+    transformation_layout = TransformationLayout()
+    transformations = model.nncf.transformation_layout().transformations
+    model = model.nncf.get_clean_shallow_copy()
+    graph = model.nncf.get_graph()
+    for command in transformations:
+        quantizer = command.fn
+
+        msg = ""
+        if not isinstance(quantizer, (SymmetricQuantizer, AsymmetricQuantizer)):
+            msg = f"Unexpected compression module on strip: {quantizer.__class__}.\n"
+        if quantizer._qspec.half_range or quantizer._qspec.narrow_range:
+            msg += "Unexpected parameters of quantizers on strip: half_range and narrow_range should be False.\n"
+        if quantizer.num_bits not in [4, 8]:
+            msg += f"Unsupported number of bits {quantizer.num_bits} for the quantizer {quantizer}.\n"
+        if len(command.target_points) > 1:
+            msg += "Command contains more than one target point."
+        if msg:
+            raise nncf.ValidationError(msg)
+
+        tp = command.target_points[0]
+        node_with_weight = graph.get_node_by_name(tp.target_node_name)
+        weight_node = get_const_node(node_with_weight, tp.input_port_id, graph)
+
+        module_name, weight_attr_name = split_const_name(weight_node.layer_attributes.name)
+        module = get_module_by_name(module_name, model)
+        original_weight = getattr(module, weight_attr_name)
+
+        original_dtype = original_weight.dtype
+        original_shape = original_weight.shape
+        original_eps = torch.finfo(original_dtype).eps
+
+        qdq_weight = quantizer.quantize(original_weight)
+        if hasattr(quantizer, "_lspec"):
+            # Reshape for group-wise quantization, implemented for classes with lora spec only
+            qdq_weight = qdq_weight.reshape(quantizer._lspec.weight_shape)
+        qdq_weight = qdq_weight.to(original_dtype)
+
+        if isinstance(quantizer, AsymmetricQuantizer):
+            input_range_safe = abs(quantizer.input_range) + quantizer.eps
+            input_low, input_range = TuneRange.apply(quantizer.input_low, input_range_safe, quantizer.levels)
```

Comment:
Please check that is not possible to reuse existed function to get zp and scales? 
It can lead to differ results on training and after strip.

Reply:
`calculate_quantized_weight` requires `WeightCompressionConfig`
`calculate_scale_zero_point` would require creation of nncf.Tensor and do unnecessary calculations with input_high, input_low, when for symmetric case it just needs `scale = quantizer.scale / abs(quantizer.level_low)`
It could lead to bigger error, because of different order of operations.